README.md

```diff
@@ -1,76 +1,62 @@
-
-All maintenance actions are executed in the blockchain network trough transactions, this is extremely helpful to refund the maintenance cost instantly as well as providing an economic incentive to those who maintain the dapp.
-
-### Cache
-
-To update the cache a script need to be executed, this script will fetch and index all the necessary information to be served in the dapp when it starts, so the dapp will only need to fetch the remaining information to the last block.
-
-Example: If the dapp is used in mainnet and the last block number is 11 million and the cache file has all the data till block number 10 million the application will get all the remaining data between block number 10 million and 11 million before starting, once it finish it will start the dapp.
-
-**Pros and Cons**: The big Pro of this is that there wont be loading times between actions while the user is using the dapp, all actions would seem to be instant, and they are because the information is already there. The Con in this approach is that if the cache is not updated often (days or weeks depending on the network) it can cause low loading times when you open the application for the first time.
-
-The cache script will gather all immutable information from the ethereum networks supported and save all of it in cache files uploaded in IPFS. This script is ran
+## Development environment setup  
+The following steps should get the app up and running:
 
-To understand a bit better what the script will do you can see `src/services/cacheService.ts`.
-
-### Configuration
-
-The configuration is even easier than the cache, it is just one file that stores all the smart contracts addresses, the tokens to be used, and the proposal templates and the calls to be recommended to be used in the new proposal.
-
-Each network in the config file has a token array, recommended calls array and templates array, if you want to add new tokens, templates and recommended calls to the dapp just add them in the config file. You only change the file when you want to change the values mentioned before.
-
-## Install
-
-Be sure you are using correct node version listed in `./package.json` (`engines.node`)
-
-`nvm use`
-
-Install all dependencies with yarn.
-
-`yarn`
-
-## Development
+1. Fork DAVI repo
+2. ``` git checkout develop ```
+3. ``` yarn ```
+4. ``` yarn dev ```
 
 The development script will start a local hardhat node, deploy all dxdao contracts with local development configuration and start the dapp with the development configuration in the port 3000.
 
-`yarn dev`
-
-To understand a bit better what the script will do you can see `scripts/dev.sh` and `scripts/deployDevContracts.ts`.
-
-## Start
-
-The script will start the dapp with the production configuration locally on port 8080 serving the folder build after running the production build.
-
-`yarn start`
-
-## Build
+### Wallet 
+- Use whichever wallet you enjoy (tested mostly with firefox and frame.sh)
```

Comment:
Should we say this? I use Brave and Metamask for every interaction and works OK.

Reply:
Oh that's a typo, thanks for spotting, was supposed to be metamask and frame.

I just realised I got those confused because they both have foxes